src/participantSummaryRow.js

```diff
@@ -92,7 +92,8 @@ const kitStatusCidToString = {
 export const baselineMouthwashSample = (participantModule) => {
   const homeMouthwashData =
     participantModule[fieldMapping.collectionDetails]?.[fieldMapping.baseline]?.[fieldMapping.bioKitMouthwash] || {};
-  const collectionTime =
+const collectionTime =
+    homeMouthwashData?.[fieldMapping.kitReceivedTime] ||
```

Comment:
Should be safe to use `homeMouthwashData[fieldMapping.kitReceivedTime]`, since `homeMouthwashData` has fallback value as `{}`.
```suggestion
    homeMouthwashData[fieldMapping.kitReceivedTime] ||
```